tests/unit/core/binarycodec/fixtures/data/data-driven-tests.json

```diff
@@ -2881,21 +2881,6 @@
       "is_negative": false,
       "exponent": -90
     },
-    {
-      "test_json": {
-        "currency": "USD",
-        "value": "100000000000000000000000000000000000000000000000000000000000000000000000000000",
-        "issuer": "rrrrrrrrrrrrrrrrrrrrBZbvji"
-      },
-      "significant_digits": 1,
-      "type_id": 6,
-      "is_native": false,
-      "mantissa": "00038D7EA4C68000",
-      "type": "Amount",
-      "expected_hex": "E7C38D7EA4C6800000000000000000000000000055534400000000000000000000000000000000000000000000000001",
-      "is_negative": false,
-      "exponent": 62
-    },
```

Comment:
Why was this test removed?

Reply:
as the calculation the IOU precision method is fixed, `100000000000000000000000000000000000000000000000000000000000000000000000000000` is not a valid IOU amount anymore, so it's failing under this method https://github.com/XRPLF/xrpl-py/blob/f2f80ccfd8385370362cc3344e4627b19995a030/xrpl/core/binarycodec/types/amount.py#L81-L109